ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -55,15 +58,25 @@ public static BlobProperties getBlobPropertiesFromStream(DataInputStream stream)
     switch (version) {
       case VERSION_1:
         toReturn = new BlobProperties(blobSize, serviceId, ownerId, contentType, isPrivate, ttl, creationTime,
-            Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID);
+            Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID, false);
         break;
-      case VERSION_2:
+      case VERSION_2: {
         short accountId = stream.readShort();
         short containerId = stream.readShort();
         toReturn =
             new BlobProperties(blobSize, serviceId, ownerId, contentType, isPrivate, ttl, creationTime, accountId,
-                containerId);
-        break;
+                containerId, false);
+      }
+      break;
+      case VERSION_3: {
```

Comment:
same here (and other places) - switch cases don't need braces? did the formatter change or something?